documentation/en/source/pages/templates-spec/builders/builders-outscale.rst

```diff
@@ -37,9 +42,9 @@ Publishing a Machine Image
 
 To publish an image, the valid keys are:
 
+* ``type`` (mandatory): default builder type for Outscale: ``Outscale``. To get the available builder type, please refer to :ref:`command-line-format`
```

Comment:
In most cases for type you have : "a string providing the cloud account type." Maybe add everywhere for consistency?